kwave/options/simulation_options.py

```diff
@@ -285,7 +285,7 @@ def option_factory(kgrid: "kWaveGrid", options: SimulationOptions):
                 options.plot_scale = [-1, 1]
 
         # replace defaults with user defined values if provided and check inputs
-        if (val := options.pml_alpha) is not None and options.pml_alpha != 'auto':
+        if (val := options.pml_alpha) is not None and not isinstance(options.pml_alpha, str):
```

Comment:
Although the conditions are not the same (previous passes if `pml_alpha != 'auto'`, now it will pass for any string), I think it is acceptable.